packages/core/src/App/Components/Elements/NotificationMessage/notification.jsx

```diff
@@ -55,7 +55,7 @@ const Notification = ({ data, removeNotificationMessage }) => {
                                     className={classNames('dc-btn', 'dc-btn--secondary', 'notification__cta-button')}
                                     to={data.action.route}
                                 >
-                                    <span className='dc-btn__text'>{data.action.text}</span>
+                                    <Text size='xxs'>{data.action.text}</Text>
```

Comment:
Should you pass `line_height` prop? The default `line_height` is `xs`, but for this component it should be `m`.

Reply:
Actually, we have changed the default, and now the default is `m` as you can see in the `text.js`